docs/deploying_clearml/clearml_server_linux_mac.md

```diff
@@ -138,9 +138,18 @@ Deploying the server requires a minimum of 4 GB of memory, 8 GB is recommended.
       ```
       sudo curl https://raw.githubusercontent.com/allegroai/clearml-server/master/docker/docker-compose.yml -o /opt/clearml/docker-compose.yml
       ```
-1. For Linux only, configure the **ClearML Agent Services**. If `CLEARML_HOST_IP` is not provided, then ClearML Agent Services uses the external public address of the ClearML Server. If `CLEARML_AGENT_GIT_USER` / `CLEARML_AGENT_GIT_PASS` are not provided, then ClearML Agent Services can't access any private repositories for running service tasks.
+1. For Linux only, configure the **ClearML Agent Services**:
+
+    * Set `CLEARML_AGENT_ACCESS_KEY` and `CLEARML_AGENT_SECRET_KEY` so the services agent can communicate with the API 
+    server. Prepare randomly-generated strings for these fields  
```

Comment:
@pollfly "so the..." -> "with confidential strings for the services agent to authenticate with the API server. Provide string values."